src/main/java/com/user_service/services/UserAuthenticationService.java

```diff
@@ -1,8 +1,14 @@
 package com.user_service.services;
 
 
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.RSAKeyProvider;
 import com.user_service.DTOs.*;
```

Comment:
You should avoid star imports: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad